doc/ale.txt

```diff
@@ -2597,7 +2597,53 @@ g:ale_cspell_use_global                               *g:ale_cspell_use_global*
 
 
 -------------------------------------------------------------------------------
-7.3. Options for languagetool                        *ale-languagetool-options*
+7.3. Options for dprint                                    *ale-dprint-options*
+
+`dprint` is a fixer for many file types, including: (java|type)script,
+json(c?), markdown, and more. See https://dprint.dev/plugins for an up-to-date
+list of supported plugins and their configuration options.
+
+The ale options for `dprint` are shared between all filetypes, so options can
+be configured only once.
```

Comment:
This is not entirely true. It is possible to add different configurations per filetype using ftplugin configurations or filetype autocommands, etc.

Reply:
Oh! I see. I copied `cspell` a bit too closely. I'll change this right now.